src/schema/objects/associated_data.yaml

```diff
@@ -6,10 +6,10 @@
 code:
   display_name: Code
   description: |
-    A directory in which to store any code used to generate the derivatives from the
-    source data.
+    A directory in which to store any code used to generate the derivatives
+    from the source data.
```

Comment:
This description feels too restrictive.

I am pretty sure we have datasets in the examples that contain also the code used to run the fMRI, EEG, ... experiments.

Also, shouldn't this way "the raw data from the source data"?

Reply:
may be better to reuse the text we already have:

https://bids-specification.readthedocs.io/en/latest/03-modality-agnostic-files.html#code

Reply:
Updated the text in the schema and added a link to the code section.